src/internal.c

```diff
@@ -15924,6 +15916,101 @@ int DoApplicationData(WOLFSSL* ssl, byte* input, word32* inOutIdx)
     return 0;
 }
 
+static void LogAlert(int type)
+{
+    (void)type;
+#ifdef DEBUG_WOLFSSL
```

Comment:
We already have a lookup in `wolfSSL_alert_type_string_long`. Can we leverage that even if pulling it out of the `OPENSSL_EXTRA` section or adding `DEBUG_WOLFSSL` around it?

Reply:
Yeah, that looks like just what I need. I didn't notice it earlier. Will change to using that. Thanks.